> @FullBinaryAlchemist Make sure to update the solution too. Also, you were supposed to run the tests as I mentioned in the issue before creating a PR.

@RandellDawson I have updated the `pwRegex` variable in the solution field. 
I'm unable to setup the environment locally so I using Gitpod. While the functionality is working by using the in-paeg browser of Gitpod , on running `npm run test:curriculum` I'm getting errors:

```
> @freecodecamp/curriculum@0.0.0-next.4 test /workspace/freeCodeCamp/curriculum
> mocha --delay --reporter progress --bail

Browserslist: caniuse-lite is outdated. Please run next command `npm update caniuse-lite browserslist`
⠏ Populate tests.
⠋ Populate tests.

  0 passing (161ms)
  1 failing

  1) Check challenges
       "before all" hook:
     Error: Failed to launch chrome!
/workspace/freeCodeCamp/curriculum/node_modules/puppeteer/.local-chromium/linux-674921/chrome-linux/chrome: error while loading shared libraries: libX11-xcb.so.1: cannot open shared object file: No such file or directory


TROUBLESHOOTING: https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md

      at onClose (node_modules/puppeteer/lib/Launcher.js:349:14)
      at Interface.helper.addEventListener (node_modules/puppeteer/lib/Launcher.js:338:50)
      at Interface.close (readline.js:379:8)
      at Socket.onend (readline.js:157:10)
      at endReadableNT (_stream_readable.js:1125:12)
      at process._tickCallback (internal/process/next_tick.js:63:19)



npm ERR! Test failed.  See above for more details.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @freecodecamp/freecodecamp@0.0.1 test:curriculum: `cd ./curriculum && npm test && cd ../`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @freecodecamp/freecodecamp@0.0.1 test:curriculum script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/gitpod/.npm/_logs/2019-10-23T19_58_48_068Z-debug.log
```
I'm having trouble resolving them, any idea on how I go around solving them ?

--- a/curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/positive-and-negative-lookahead.english.md
+++ b/curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/positive-and-negative-lookahead.english.md
@@ -60,6 +60,9 @@ tests:
     testString: assert(!pwRegex.test("1234"));
   - text: Your regex should not match <code>"8pass99"</code>
     testString: assert(!pwRegex.test("8pass99"));
+  - text: Your regex should not match <code>"12abcde"</code>
+    testString: assert(!pwRegex.test("12abcde"));
+
 
 
 ```

--- a/curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/positive-and-negative-lookahead.english.md
+++ b/curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/positive-and-negative-lookahead.english.md
@@ -88,7 +88,7 @@ let result = pwRegex.test(sampleWord);
 
 
 ```js
-var pwRegex = /^(?=\w{6})(?=\D*\d{2})/;
+var pwRegex = /^(?=\w{6})(?=\D+\d{2})/;
 ```
 
 </section>

